msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.HotRestart.targets

```diff
@@ -246,9 +246,9 @@
 			_CreateHotRestartCachedBundle;
 			_UnpackLibraryResources;
 			_ComputeHotRestartBundleContents;
-			_CodesignHotRestartAppBundle;
 			_CopyHotRestartBundleResources;
 			_CopyFilesToHotRestartSignedAppDirContents;
+			_CodesignHotRestartAppBundle;
```

Comment:
I remember I had to change the order of that target because it was causing unwanted behaviors. I don't know if you checked that as part of this changes. There are more comments in my PR: https://github.com/xamarin/xamarin-macios/commit/495ebc9ffd9d67220deb9394f5ddaee7f6019b16

The related ones are: 

`- Ensure the _CodesignHotRestartAppBundle target is executed before the copy of the content files and not after: Hot Restart content files doesn't affect the code signing, so they don't need to be copied before the signing process. Copying the content files before the code sign was causing unwanted behaviors and errors since the code sign logic will try to clear the signing folder before the execution, to avoid mixing old and new content`

Reply:
@mauroa I checked your changes, and now we're signing in the middle of copying files: files that must be signed are copied, then we sign, then content files are copied: https://github.com/rolfbjarne/xamarin-macios/blob/3a2860296c2fc742f16c3d3a42e3fc0e1ee6c7df/msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.HotRestart.targets#L249-L253

That's the intention at least - but how do I test that your scenario still works?

Reply:
Right, I didn't realize the order was not exactly the same as the one before my changes. My bad I didn't add the bug link to my PR, I'll see If I can track it somehow

Reply:
@mauroa were you able to find the bug link?